pom.xml

```diff
@@ -24,6 +24,7 @@
     <pentaho-scheduler-plugin.version>10.2.0.0-SNAPSHOT</pentaho-scheduler-plugin.version>
     <com.github.spotbugs.annotations.version>4.2.3</com.github.spotbugs.annotations.version>
     <platform.version>10.2.0.0-SNAPSHOT</platform.version>
+    <maven-antrun-plugin.version>1.7</maven-antrun-plugin.version>
```

Comment:
Can we use a more recent version (e.g. 3.1.0)?

Reply:
Done 👍